### PR TITLE
convert to js with babel

### LINKED ISSUE #33: Refactor into plain JS (remove typescript)
I want to merge this into the main repo, and introducing typescript into the main repo is a bad idea at this point.

### PR DESCRIPTION
**Pull request by psimyn**
closes https://github.com/storybooks/addon-backgrounds/issues/33

seems to work

### CODE REVIEW

File: src/BackgroundPanel.js [hunk 1]
```diff
@@ -62,14 +45,20 @@ const Instructions = () => (
...
-  state = { backgrounds: [] };
-
+export default class BackgroundPanel extends React.Component {
   constructor(props) {
     super(props);
 
-    this.props.channel.on("background-set", backgrounds => {
+    // A channel is explicitly passed in for testing
+    if (this.props.channel) {
+      this.channel = this.props.channel;
```
Comment (psimyn) [inline src/BackgroundPanel.js]: `props.channel` is undefined on initial render. not sure why 😖

**Comment by ndelangen** [comment]
Can you resolve the merge conflicts? 👍

File: package.json [hunk 2]
```diff
@@ -37,39 +36,46 @@
...
     "babel-plugin-transform-react-constant-elements": "^6.9.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.6",
     "babel-preset-es2015-loose": "^8.0.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
     "babel-traverse": "^6.14.0",
     "coveralls": "^2.11.12",
-    "enzyme": "^2.4.1",
+    "enzyme": "^2.9.1",
+    "eslint": "^3.19.0",
```
Comment (ndelangen) [inline package.json]: you're adding all these eslint dependencies but there's no lint script?

Looks like these rules are similar or the same as the main repo 👍

File: package.json [hunk 3]
```diff
@@ -37,39 +36,46 @@
...
     "redbox-react": "^1.3.0",
-    "source-map-loader": "^0.2.1",
-    "ts-loader": "^0.8.2",
-    "tslint": "^3.15.1",
-    "typescript": "^1.8.10"
+    "source-map-loader": "^0.2.1"
   },
   "dependencies": {
-    "object-assign": "^4.1.0"
+    "babel-cli": "^6.24.1"
```
Comment (ndelangen) [inline package.json]: Is `babel-cli` a runtime dependency? If it's only required for build, it should be a devDependency?

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (psimyn) [inline package.json]: copied them from there, but there was hundreds of failing lints. I'll do the TS=>JS first then run prettier over it in another MR

**Comment by psimyn** [comment]
@ndelangen updated and fixed 👍

**Review by ndelangen — APPROVED** [review]

**Pull request merged**